Dockerfile

```
@@ -14,15 +15,36 @@ RUN rm src/lib.rs
 COPY src/* src/
 # build the executable
 RUN cargo build --release
+RUN mkdir -p ${wd}/bin
+COPY bin/generate-registry.sh ${wd}/bin
```

I _think_ you might be able to just do:

```suggestion
COPY bin/generate-registry.sh ${wd}/bin
```

---

If it's `bin/` (with a trailing slash) it does work, thank you!